bleachbit/Action.py

```diff
@@ -71,7 +71,7 @@ def expand_multi_var(s, variables):
     for var_set in vars_product:
         ms = s  # modified version of input string
         for var_key, var_value in var_set.items():
-            sub = '$$%s$$' % var_key
+            sub = '{}{}{}'.format(var_format, var_key, var_format)
```

Comment:
This string formatting change is okay. It's longer, though. What is the benefit you see?

Reply:
Thank you for taking time to review this.

The benefit here would be that if `$$` needed to be changed to some other token, changing the string value from the `var_format` constant variable is only needed. It's just a maintenance improvement.

To make it shorter, `var_format` could be renamed to `var_fmt` or something more appropriate. Since it's a constant variable, it probably needs to be all caps to notate that it's a constant as well.